sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/models/PartitionPumpOptions.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs.models;
+
+import java.util.Objects;
+
+/**
+ * Set of options that can be specified when build each partition pump.
+ */
+public class PartitionPumpOptions {
```

Comment:
Add PartitionPumpOptions to let user configure partition pumps.  And pass this to PartitionPumpManager.
Currently the configuration is applied to all partition pumps. Should we need have ability to configure for each partition?